libs/client-shared/src/lib/components/button/button.component.scss

```diff
@@ -73,6 +73,11 @@
       color: variables.$white;
     }
   }
+
+  &:focus.asset-sg-active,
```

Comment:
Pseudo-classes need to be written after classes (does this even work this way?):
```suggestion
  &.asset-sg-active:focus,
```

Reply:
yep, worked 👯